lib/list-unix.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+var async = require('async');
+var exec = require('child_process').exec;
+var fs = require('fs');
+var path = require('path');
+
+function udevParser(output) {
+  var udevInfo = {};
+  var lines = output.split('\n');
+  lines.forEach(function(line) {
+    if (!line || line.trim() === '') {
+      return;
+    }
+    var line_parts = line.trim().split('=');
+    var key = line_parts[0].trim().toLowerCase();
+    udevInfo[key] = line_parts[1].trim();
+  });
+
+  var pnpId;
+  if (udevInfo.devlinks) {
+    udevInfo.devlinks.split(' ').forEach(function(path) {
+      if (path.indexOf('/by-id/') === -1) { return }
+      pnpId = path.substring(path.lastIndexOf('/') + 1);
+    });
+  }
+
+  var vendorId = udevInfo.id_vendor_id;
+  if (vendorId && vendorId.substring(0, 2) !== '0x') {
+    vendorId = '0x' + vendorId;
+  }
+
+  var productId = udevInfo.id_model_id;
+  if (productId && productId.substring(0, 2) !== '0x') {
+    productId = '0x' + productId;
```

Comment:
Still haven't seen any examples in all of the docs, tutorials, articles, SO questions, etc. that show these values having `0x` prefix